source/agora/utils/PrettyPrinter.d

```diff
@@ -172,22 +172,22 @@ private struct PubKeyFmt
     {
         try
         {
-            // Public keys are 56 characters, only take the first 8 and last 4
-            // Only format `0xABCDEFGH..JKLM`
-            enum StartUntil = 8;
-            enum EndFrom    = 56 - 4;
-            size_t count;
+            // Public keys are 63 characters, only take the first 12 and last 4
+            // Only format `0xboa1acdefghi..6789`
```

Comment:
```suggestion
            // Only format `boa1acdefghi..6789`
```

Reply:
Actually I wonder if we should just not print the HRP ?

Reply:
I thought about that also. Can't we use it as identifying the network like bitcoin's use case, like `bt`, `tb`?  Anyway, we can remove that now and restore it later if needed.